lib/jekyll/configuration.rb

```diff
@@ -113,6 +113,10 @@ def source(override)
       override['source'] || self['source'] || DEFAULTS['source']
     end
 
+    def in_source(override, path)
```

Comment:
This method feels weird here. Don't we have a `Utils` namespace that this could go to instead?